src/NServiceBus.Core/Unicast/Behaviors/LoadHandlersConnector.cs

```diff
@@ -18,7 +17,7 @@ public LoadHandlersConnector(MessageHandlerRegistry messageHandlerRegistry)
 
         public override void Invoke(LogicalMessageProcessingStageBehavior.Context context, Action<HandlingStageBehavior.Context> next)
         {
-            var handlerTypedToInvoke = messageHandlerRegistry.GetHandlerTypes(context.MessageType).ToList();
+            var handlerTypedToInvoke = messageHandlerRegistry.GetHandlersFor(context.MessageType).ToList();
```

Comment:
fixing typo. I already talked with Daniel about some namings. The returned value is a `IEnumerable<MessageHandler>`. I think we should first think about renaming the `MessageHandler` to something more accurate (since MessageHandler.Instance will contain the handler instance).